ci/nixpkgs-build/build-context/entrypoint.sh

```diff
@@ -5,7 +5,7 @@ set -e -o pipefail -u -x
 git clone --depth=1 https://github.com/NixOS/nixpkgs.git .
 
 source_hash=$(nix-prefetch-url --unpack https://github.com/VirtusLab/git-machete/archive/$GIT_REVISION.tar.gz)
-version=$(curl https://raw.githubusercontent.com/VirtusLab/git-machete/$GIT_REVISION/git_machete/__init__.py | cut -d\' -f2)
+version=$(curl https://raw.githubusercontent.com/VirtusLab/git-machete/$GIT_REVISION/git_machete/__init__.py | cut -d\' -f2) #NOTE: NOT SURE HOW THIS CAN BE DONE
```

Comment:
Need some help with this one.

Reply:
![goncern](https://user-images.githubusercontent.com/3383210/157290540-a9f9722d-b3bf-494b-8cfb-51796b8e5403.png)

Reply:
A fix worse than the problem itself would be to check out the repo at `$GIT_REVISION` and then run `python3 setup.py --version` on that repo...

Reply:
Let's leave as is! the current approach is simpler than the alternative